node/mock-toolrunner.ts

```diff
@@ -19,6 +19,7 @@ export interface IExecSyncOptions {
     cwd?: string;
     env?: { [key: string]: string | undefined };
     silent?: boolean;
+    hideCommand?: boolean;
```

Comment:
Good point @anatolybolshakov. It might be cleaner if we move `hideCommand` to `IExecOptions` instead, which extends `IExecSyncOptions`. What do you think?

Some other options which I'm happy to implement if you guys prefer:

1. Optional constructor parameter to `ToolRunner` class
2. Optional parameter for `execWithPiping`, `exec`, and `execSync` methods

Reply:
I have moved `hideCommand` parameter to `IExecOptions` interface. It is now only implemented in `exec` & `execWithPiping` methods, but not `execSync`, which works for me.

Let me know if you're cool with that. Otherwise, happy to implement any other approach mentioned above.

Reply:
@anatolybolshakov & @mmrazik could you guys please review when you have a moment? Thanks!